### PR TITLE
Datepicker reopens after clicking on the input when the datepicker is already open

### PR DESCRIPTION
**Pull request by liamhubers**
We're having some problems with people clicking on the input to close to datepicker ( mainly on mobile ) however right now the datepicker will close and reopen directly after.

### CODE REVIEW

**Comment by liamhubers** [comment]
Not quite sure why I'm getting conflicts..

**Comment by martijnrusschen** [comment]
Did you try to rebase with the latest master? Looks like you're still running against an old master.

**Comment by liamhubers** [comment]
Fixed it :)

File: src/datepicker.jsx [hunk 1]
```diff
@@ -110,10 +109,18 @@ var DatePicker = React.createClass( {
...
+    if (!this.state.virtualFocus) {
+      return this.setState({
+        focus: true,
+        virtualFocus: true
+      });
+    }
+    this.setState({ virtualFocus: false });
+  },
+
+  onClearClick: function( event ) {
```
Comment (rafeememon) [inline src/datepicker.jsx]: Looks like this wasn't merged properly

Comment (liamhubers) [inline src/datepicker.jsx]: Merged correctly, seeing anything weird I'm not seeing?

Comment (rafeememon) [inline src/datepicker.jsx]: This adds a duplicated definition of `onClearClick`, which is defined below this diff

Comment (liamhubers) [inline src/datepicker.jsx]: Nice catch, thanks!

**Pull request merged**